src/main/scala/uk/gov/hmrc/ct/box/ValidatableBox.scala

```diff
@@ -238,6 +240,13 @@ trait ValidatableBox[T <: BoxRetriever] {
     }
   }
 
+  def validateStringMaxLength(boxId: String, value: String, max: Int): Set[CtValidation] = {
```

Comment:
I think the one you are talking about is a different method, as it check for a string length inside a range, so min and max, and returns both min and max in the error and a different error code